ipn/ipnlocal/local.go

```diff
@@ -852,6 +852,11 @@ func (b *LocalBackend) setClientStatus(st controlclient.Status) {
 			}
 		}
 
+		// Ensure that we also fire this timer if our own node key expires.
+		if selfExpiry := st.NetMap.SelfNode.KeyExpiry; !selfExpiry.IsZero() && selfExpiry.Before(nextExpiry) {
```

Comment:
SelfNode can't be nil at this point? I thought there were cases it could be but forget where.

Reply:
At least in the tests, it appears so!